Index: lesson/Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package lesson;\r\n\r\npublic class Main {\r\n    public static void main(String[] args) {\r\n\r\n    }\r\n}\r\n
===================================================================
diff --git a/lesson/Main.java b/lesson/Main.java
--- a/lesson/Main.java	
+++ b/lesson/Main.java	
@@ -1,7 +1,22 @@
 package lesson;
 
+import static lesson.Lesson_5.*;
+
 public class Main {
     public static void main(String[] args) {
-
+        printThreeWords(); /*1 задание*/
+        checkSumSign(); /*2 задание*/
+        printColor(); /*3 задание*/
+        compareNumbers(); /*4 задание*/
+        System.out.println(isSumInRange()); /*5 задание*/
+        printPositiveOrNegative(); /*6 задание*/
+        System.out.println(isNegative()); /*7 задание*/
+        printStringMultipleTimes(); /*8 задание*/
+        System.out.println(isLeapYear()); /*9 задание*/
+        invertBinaryArray(); /*10 задание*/
+        fillArray(); /*11 задание*/
+        multiplyElements(); /*12 задание*/
+        fillDiagonal(); /*13 задание*/
+        createArray(); /*14 задание*/
     }
 }
